Runtime/Unity.ProBuilder.asmdef

```diff
@@ -2,15 +2,22 @@
     "name": "Unity.ProBuilder",
     "references": [
         "Unity.ProBuilder.Poly2Tri",
-        "Unity.ProBuilder.KdTree"
+        "Unity.ProBuilder.KdTree",
+        "Unity.RenderPipelines.HighDefinition.Runtime"
```

Comment:
Does this also need to reference Universal Render Pipeline, or is that implicit?

Reply:
Not at the moment. This PR is solely for HDRP.
The old codepath still work with URP so I kept the way it is. At some point, we'll have to give it a deeper look.

Reply:
After a quick look, URP doesn't really work. It seems to work but in many cases it won't work as we draw way more on the lookup texture than just our picking data. We filed a ticket for that.